fetch.bs

```diff
@@ -414,38 +414,92 @@ each other by 0x2C 0x20, in order.
 
 <hr>
 
-<p id=simple-header>A <dfn export>CORS-safelisted request-header</dfn> is a <a for=/>header</a>
-whose <a for=header>name</a> is a <a>byte-case-insensitive</a> match for one of
+<p id=simple-header>To determine whether a <a for=/>header</a> <var>header</var> is a
```

Comment:
Did you intent to produce text that says "header header" with the first header linked?

Reply:
Oh, I guess this is just `<type> <var name>`.  Its just confusing to me when the type and variable name are identical.